draft-ietf-quic-transport.md

```diff
@@ -124,10 +124,53 @@ provides:
 
 * Authenticated and encrypted header and payload
 
-QUIC uses UDP as a substrate to avoid requiring changes to legacy client
-operating systems and middleboxes.  QUIC authenticates all of its headers and
-encrypts most of the data it exchanges, including its signaling, to avoid
-incurring a dependency on middleboxes.
+QUIC establishes a connection, which is a stateful interaction between a client
+and server. The primary purpose of a connection is to support the exchange of
+data by an application protocol.
+
+Streams are means by which an application protocol exchanges information.
+Streams are ordered sequences of bytes.  Two types of stream can be created:
+bidirectional streams allow both endpoints to send data; unidirectional streams
+allow a single endpoint to send.  A credit-based scheme is used to limit stream
+creation and to bound the amount of data that can be sent.
+
+The QUIC handshake combines negotiation of cryptographic and transport
+parameters.  The handshake is structured to permit the exchange of application
+data as soon as possible.  This includes an option for clients to send data
+immediately (0-RTT), which might require prior communication to enable.
+
+QUIC connections are not strictly bound to a single network path.  Connection
+migration uses connection identifiers to allow connections to transfer to a new
+network path.
+
+Frames are used in QUIC to communicate between endpoints.  One or more frames
+are assembled into packets.  QUIC authenticates all packets and encrypts as much
+as is practical.  QUIC packets are carried in UDP datagrams to better facilitate
+deployment in existing systems and networks.
+
+{{fig-hs}} shows a simplied handshake and the exchange of packets and frames
+that are used to advance the handshake.  Exchange of application data during the
+handshake is enabled where possible, shown with a '*'.
```

Comment:
This seems a bit out of place. It would be more useful if it included data exchange and connection termination. Otherwise, I would suggest removing it entirely (this para and the text below as well).